staging/persistent-volume-provisioning/README.md

```diff
@@ -188,6 +189,10 @@ Please note that, the value for this parameter cannot contain `_` in storageclas
 
 *`snapfactor`: Dynamically provisioned volume's thinpool size can be configured with this parameter. The value for the parameter should be in range of 1-100, this value will be taken into account while creating thinpool for the provisioned volume. This is an optional parameter with default value of 1.
 
+* `customepnameprefix` : By default dynamically provisioned volumes has an endpoint and service created with the naming schema of `glusterfs-dynamic-<PVC UUID` format. With this option present in storageclass, an admin can now prefix the desired endpoint from storageclass. If `customepnameprefix` storageclass parameter is set, the dynamically provisioned volumes will have an endpoint and service created in below format where `-` is the field separator/delimiter:
+
+`customepnameprefix-<PVC UUID>`
```

Comment:
you need to indent this a bit (maybe 6 characters), it renders outside the ordered list item's paragraph.